mail_tracking/models/mail_message.py

```diff
@@ -86,6 +88,11 @@ def _get_error_description(self, tracking):
 
     def tracking_status(self):
         """Generates a complete status tracking of the messages by partner"""
+        db_aliases = set(map(
+            lambda x: x['alias_name'],
+            self.env['mail.alias'].search_read([
+                ('alias_name', '!=', False),
+            ], ['alias_name'])))
```

Comment:
I'll try to help save some loops here because I see big chances of performance impact.

```suggestion
        catchall_domain = self.env['ir.config_parameter'].get_param('mail.catchall.domain')
        db_aliases = []
        if catchall_domain:
            db_aliases = [
                "%s@%s" % (rec.alias_name, catchall_domain)
                for self.env['mail.alias'].search([
                    ('alias_name', '!=', False),
                ])
            ]
```

Reply:
Better than Odoo eh... we see the domain :P thx